specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/storage.json

```diff
@@ -4313,12 +4313,58 @@
         "filters": {
           "$ref": "#/definitions/BlobInventoryPolicyFilter",
           "description": "An object that defines the filter set."
+        },
+        "format": {
+          "type": "string",
+          "description": "This is a required field, it specifies the format for the inventory files.",
+          "enum": [
+            "Csv",
+            "Parquet"
+          ],
+          "x-ms-enum": {
+            "name": "format",
+            "modelAsString": true
+          }
+        },
+        "schedule": {
+          "type": "string",
+          "description": "This is a required field. This field is used to schedule an inventory formation.",
+          "enum": [
+            "Daily",
+            "Weekly"
+          ],
+          "x-ms-enum": {
+            "name": "schedule",
+            "modelAsString": true
+          }
+        },
+        "objectType": {
+          "type": "string",
+          "description": "This is a required field. This field specifies the scope of the inventory created either at the blob or container level.",
+          "enum": [
+            "Blob",
+            "Container"
+          ],
+          "x-ms-enum": {
+            "name": "objectType",
+            "modelAsString": true
+          }
+        },
+        "schemaFields": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "This is a required field. This field specifies the fields and properties of the object to be included in the inventory. Valid values for this field for the blob object type include Name, Creation-Time, Last-Modified, Content-Length, Content-MD5, BlobType, AccessTier, AccessTierChangeTime, Expiry-Time, hdi_isfolder, Owner, Group, Permissions, Acl, Snapshot, VersionId, IsCurrentVersion, Metadata, LastAccessedTime. Valid values for container object type include Name, Last-Modified, Metadata, LeaseStatus, LeaseState, LeaseDuration, PublicAccess, HasImmutabilityPolicy, HasLegalHold."
         }
       },
       "required": [
-        "filters"
```

Comment:
As filter is not required now, will blobtype still be required for filter?
It seems not much valuable to make blobtype required in filter, as we can inventory blobs with all types by default.

Reply:
blobType is required when objectType is Blob. I have updated the Descrtiption